apns_csr/builder.py

```diff
@@ -14,7 +14,7 @@
 def _get_remote_cert(url):
 # TODO(dustin): There's an encoding problem with requests.get(url).text, and partial data when we do requests.get(url).raw.read() .
 # TODO(dustin): Cache/store this locally, or we'll be out of luck if their systems are unavailable.
-    data = urllib2.urlopen(url).read()
+    data = urllib.request.urlopen(url).read()
     if not data:
```

Comment:
Let's do `if data is not None:` here. We should prefer to be specific. 

`urlopen()` may return `None` (most errors will raise here; it may return `None` is if no handler was found for the scheme). So, please check if urlopen() has returned `None` before attempting to call `read()` on it. `read()`, on the other hand, will either return data or raise so there's no point in checking that return value.